### PR TITLE
Renku version upgrade and fixing python requirements

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Two main sets of functionalities are provided:
 
 ## `display` command
 
-The CLI command outputs a graphical representation of the graph as a PNG image.
+This CLI command outputs a graphical representation of the whole project knowledge graph, as a PNG image. The graph displayed will be containing also the ontology (if this is loaded via the configuration, as explained later within this document).
```
Comment (volodymyrss) [inline README.md:20]: these changes do not correspond to the title. Please either change the title or move changes to another PR.

**Review by volodymyrss — CHANGES_REQUESTED** [review]

File: renkugraphvis/__init__.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 import bs4
 
 import renkugraphvis.graph_utils as graph_utils
+from .io_utils import gitignore_file
```
Comment (volodymyrss) [inline renkugraphvis/__init__.py:30]: not used?

File: README.md [hunk 3]
```diff
@@ -124,7 +122,7 @@ An ontology can be integrated into the graph. By doing so, it is possible to gai
 ***
 # Plugin requirements and installation
```
Comment (volodymyrss) [inline README.md:126]: probably major version compatibility is sufficient? assuming renku follows semver.

File: setup.py [hunk 4]
```diff
@@ -21,8 +21,7 @@
     'pydotplus',
     'rdflib',
     'bs4',
-    'renku==2.6.0',
-    'nb2workflow>=1.3.41',
+    'renku==2.7.0',
```
Comment (volodymyrss) [inline setup.py:24]: would this work?

```suggestion
    'renku>=2.7,<3',
```

File: README.md (hunk 3 above)
Comment (burnout87) [reply README.md:126]: I assume you meant to adapt the paragraph so to mention that a minimum version of renku is needed?
Or also the title?

**Review by volodymyrss — APPROVED** [review]
